app/components/elements/forms/text_field_component.html.erb

```diff
@@ -1,6 +1,6 @@
-<div>
+<div class="mb-4 field-container" <%= 'hidden' if hidden %>>
```

Comment:
Can you do `<%= tag.div class: 'mb-4 field-container', hidden: do %>`?

(I'd suggest we be consistent with this style when setting html tag attributes.)